spec/labels_pr_checker_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+
+module Danger
+  describe Danger::LabelsPRChecker do
+    it 'should be a plugin' do
+      expect(described_class.new(nil)).to be_a Danger::Plugin
+    end
+
+    describe 'with Dangerfile' do
+      before do
+        @dangerfile = testing_dangerfile
+        @plugin = @dangerfile.labels_pr_checker
+      end
+
+      it 'returns a warning when a PR does not have labels' do
+        pr_labels = []
+        allow(@plugin.github).to receive(:pr_labels).and_return(pr_labels)
+
+        @plugin.check_labels
+
+        expect(@dangerfile.status_report[:warnings].count).to eq 1
```

Comment:
Similar to my comment on your other PR, I'd suggest your `expect` to check the actual content of the status_report, and especially the warning messages they contain, rather than assert on their count only.